pkgs/development/tools/gotemplate/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gotemplate";
+  version = "3.7.2";
+
+  src = fetchFromGitHub {
+    owner = "coveooss";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-1jyTZBkt+nN52jgs5XZN22zw33i0yENDc4cW/Y1Lidc=";
+  };
+
+  vendorHash = "sha256-WW7X3rURdvmSjbtRkeLoicsiqxsMED5el+Jl5yYk7hA=";
+
+  meta = with lib; {
+    description = "CLI for go text/template";
+    changelog = "https://github.com/coveooss/enola/gotemplate/tag/v${version}";
```

Comment:
```suggestion
    changelog = "https://github.com/coveooss/gotemplate/releases/tag/v${version}";
```

Reply:
Oh my... thanks.
(I'm squashing the commits)